apis/v1alpha1/generator.yaml

```diff
@@ -173,3 +182,26 @@ resources:
       ignore: true
     update_operation:
       custom_method_name: customUpdateHostedZone
+  HealthCheck:
+    renames:
+      operations:
+        GetHealthCheck:
+          input_fields:
+            HealthCheckId: Id
+    exceptions:
+      terminal_codes:
+      - TooManyHealthChecks
+      - HealthCheckAlreadyExists
+      - InvalidInput
+      - HealthCheckInUse
```

Comment:
Terminal errors/conditions are typically errors that we will never be able to recover from. Maybe  `TooManyHealthChecks` and `HealthCheckAlreadyExists` aren't terminal for this case?

Reply:
Removed them